AsyncDisplayKit/ASCellNode.mm

```diff
@@ -119,8 +119,11 @@ - (void)layoutDidFinish
 
 - (void)__setNeedsLayout
 {
-  CGSize oldSize = self.calculatedSize;
+  CGSize oldSize = self.bounds.size;
```

Comment:
Do we require this is called on the main thread?  If so, maybe we should assert that.  If not, maybe we should read self.threadsafeBounds.size